<launch>
<node name="path_publisher_nokov" pkg="mpc_control" type="path_publisher_nokov">
<param name="bag_file" value = "$(find mpc_control)/src/robot_1_path.bag"/>
<param name="nokov_topic" value = "robot_1/nokov"/>
</node>

<node name="rviz" pkg="rviz" type="rviz" args="-d $(find mpc_control)/cfg/rviz_nokov.rviz" />
</launch>
